packages/devextreme/js/__internal/grids/grid_core/editor_factory/m_editor_factory.ts

```diff
@@ -103,7 +115,11 @@ export class EditorFactory extends ViewControllerWithMixin {
   }
 
   _needHideBorder($element) {
-    return $element.hasClass(EDITOR_INLINE_BLOCK);
+    const rowsViewElement = this.getView('rowsView').element();
+    const isRowsView = $element.closest(rowsViewElement).length > 0;
+    const isEditing = this.getController('editing').isEditing();
+
+    return $element.hasClass(EDITOR_INLINE_BLOCK) || isRowsView && !isEditing;
```

Comment:
Minor comment.

Let's add the brackets to make this bool expression more obvious:
```suggestion
    return $element.hasClass(EDITOR_INLINE_BLOCK) || (isRowsView && !isEditing);
```